pkg/compliance/clustercompliancereport.go

```diff
@@ -86,6 +87,13 @@ func (r *ClusterComplianceReportReconciler) generateComplianceReport(ctx context
 				}
 				log.Info("Cluster compliance report written", "path", reportPath)
 
+				// Send webhook notification for compliance report
+				if r.Config.WebhookBroadcastURL != "" {
+					if err := webhook.SendWebhookReport(&report, r.Config); err != nil {
+						log.Error(err, "Failed to send compliance report via webhook")
+					}
+				}
+
```

Comment:
Thanks for the PR! Can you add a small unit test that would exercise this? One idea to do so would be to create an http test server and then pass its URL to this function. That way we can assert the results within the test server.

Reply:
@simar7 added a test in https://github.com/aquasecurity/trivy-operator/pull/2683/commits/3ba57d23dde20f542cc480b386d1b7cb0f8b098e; plz review